src/viam/trajex/totg/trajectory.cpp

```diff
@@ -1125,11 +1182,19 @@ trajectory trajectory::create(class path p, options opt, integration_points poin
                         break;
                     }
 
-                    // Candidate point is feasible - accept it and advance
+                    // Candidate point is feasible - accept it and advance. Observe the tricky behavior around the
+                    // acceleration.  We need to update the acceleration of the current point: that's the acceleration
+                    // we selected at our current location in the phase plane in order to get to the next point. That
+                    // new point now has an indeterminate acceleration, which will not be computed until the next point,
+                    // etc.
+                    //
                     // Note: Don't call maybe_on_trajectory_extended() here - forward points are tentative
                     // and may be discarded during backward integration. Only finalized segments trigger that callback.
-                    integration_point next_point{
-                        .time = current_point.time + traj.options_.delta, .s = next_s, .s_dot = next_s_dot, .s_ddot = s_ddot_max};
+                    integration_point next_point{.time = current_point.time + traj.options_.delta,
+                                                 .s = next_s,
+                                                 .s_dot = next_s_dot,
+                                                 .s_ddot = arc_acceleration{0.0}};
```

Comment:
We are setting this to zero since we do not know it's true value yet correct?
With respect to this comment
```
                    // Candidate point is feasible - accept it and advance. Observe the tricky behavior around the
                    // acceleration.  We need to update the acceleration of the current point: that's the acceleration
                    // we selected at our current location in the phase plane in order to get to the next point. That
                    // new point now has an indeterminate acceleration, which will not be computed until the next point,
                    // etc.
```

It does make sense, but still something leaves me feeling wary.

a = delta v / delta t == v_f - v_i / t_f - t_i which just gives us the average acceleration.

Reply:
Filed https://viam.atlassian.net/browse/RSDK-13002